pkg/api/audit.go

```diff
@@ -63,30 +61,18 @@ func StartAuditTrail(clusterID string, config core.CADFConfiguration) {
 		s := make(chan cadf.Event, 20)
 		eventSink = s
 
-		if config.RabbitMQ.Username == "" {
-			config.RabbitMQ.Username = "guest"
-		}
-		if config.RabbitMQ.Password == "" {
-			config.RabbitMQ.Password = "guest"
-		}
-		if config.RabbitMQ.Hostname == "" {
-			config.RabbitMQ.Hostname = "localhost"
-		}
-		if config.RabbitMQ.Port == 0 {
-			config.RabbitMQ.Port = 5672
-		}
 		rabbitURI := url.URL{
 			Scheme: "amqp",
-			Host:   net.JoinHostPort(config.RabbitMQ.Hostname, strconv.Itoa(config.RabbitMQ.Port)),
-			User:   url.UserPassword(config.RabbitMQ.Username, string(config.RabbitMQ.Password)),
+			Host:   net.JoinHostPort(osext.GetenvOrDefault("LIMES_AUDIT_RABBITMQ_HOSTNAME", "localhost"), osext.GetenvOrDefault("LIMES_AUDIT_RABBITMQ_PORT", "5672")),
+			User:   url.UserPassword(osext.GetenvOrDefault("LIMES_AUDIT_RABBITMQ_USERNAME", "rabbitmq"), os.Getenv("LIMES_AUDIT_RABBITMQ_PASSWORD")),
```

Comment:
This does not match the documented behavior:
```suggestion
			User:   url.UserPassword(osext.GetenvOrDefault("LIMES_AUDIT_RABBITMQ_USERNAME", "guest"), osext.GetenvOrDefault("LIMES_AUDIT_RABBITMQ_PASSWORD", "guest")),
```